test/api/v3/unit/libs/payments.test.js

```diff
@@ -55,6 +61,21 @@ describe('payments/index', () => {
       expect(difference - 60).to.be.lessThan(3); // the difference is approximately two months, +/- 2 days
     });
 
+    it('data.nextBill is undefined', () => {
```

Comment:
Uhg, looking at the rest of the file, this test file did not model good it blocks.